jib-core/src/main/java/com/google/cloud/tools/jib/builder/BuildConfiguration.java

```diff
@@ -178,6 +198,63 @@ public BuildConfiguration build() {
           throw new IllegalStateException(errorMessage.toString());
       }
     }
+
+    /**
+     * TODO: Move this to a class in frontend
```

Comment:
Let's file a bug to take care of these TODOs

Reply:
https://github.com/GoogleContainerTools/jib/issues/442